tests/scripts/thread-cert/node.py

```diff
@@ -1179,7 +1179,7 @@ def set_next_mlr_response(self, status: int):
         self.send_command(cmd)
         self._expect_done()
 
-    def register_multicast_listener(self, *ipaddrs: Union[ipaddress.IPv6Address, str], timeout=None):
+    def register_multicast_listener(self, *ipaddrs: List[Union[ipaddress.IPv6Address, str]], timeout=None):
```

Comment:
```suggestion
    def register_multicast_listener(self, *ipaddrs: Union[ipaddress.IPv6Address, str], timeout=None):
```
The original typing is correct.